src/core_plugins/kibana/common/field_formats/types/_numeral.js

```diff
@@ -26,6 +26,15 @@ export class Numeral extends FieldFormat {
       }
 
       getParamDefaults() {
+        const language = this.getConfig('format:language');
+        if (language !== numeral.language()) {
+          if (language !== 'en') {
+            const languageData = require(`@elastic/numeral/languages/${language}`);
```

Comment:
@spalger @kobelb 
Do you see a better way to load language data ?
Ideally, I would like to load statically all languages data to store them in numeral language map.
I try `import from @elastic/numeral/languages';` but it doesn't work.

Reply:
@fbaligand feel free to add the ability to do so at https://github.com/elastic/numeral-js

Reply:
@spalger 
One option for that is to use v3.0.0 that is already released. This version is compatible with static `import`. We can change the package.json file to depend on this version.
Are you OK to depend on this version ? Or do you consider this major version implies so much breaking changes ?

Reply:
@spalger 
I just commited migration to @elastic/numeral v3.0.0
I had no particular problem, it works fine !
And now I can `import` successfully all languages using a static way.
And so, I moved the code to `_convert` method as you wish.

Reply:
@spalger 
What do you think about ?

Reply:
Sorry, I should have deprecated `@elastic/numeral@3.0.0` [months ago](https://github.com/elastic/kibana/issues/7543#issuecomment-288508387). Unfortunately the way that numeral transforms format strings in v3 is not backwards compatible with v2. This means that in order to upgrade to numeral v3 we would either have to track down all the differences between the two versions, or users to recreate/double-check all of their numeric field formatters. We're currently not planning to do either, so this isn't going to work. I've updated [elastic/numeral-js](https://github.com/elastic/numeral-js) to reflect the current state of the package and changed https://github.com/elastic/numeral-js/blob/kibana-fork/languages.js

The file now exports the languages as an array. each item has an `id`, `name`, and `lang` property.
  - `id`: the filename of the lang
  - `name`: the english description of the language, [extracted from each language file](https://github.com/elastic/numeral-js/blob/kibana-fork/languages/de.js#L3)
  - `lang`: the value that should be passed to `numeral.language(id, lang)`

The file doesn't automatically register the languages, maybe we want it to do that, but it should be a suitable replacement for the current languages import, right? To test it out install `@elastic/numeral` from github:

```sh
npm install https://github.com/elastic/numeral-js#kibana-fork
```

Reply:
OK.
Thanks for all your work to transform "languages.js" in a way it can be imported easily !
So I'm ready to cancel my upgrade to 3.0.0 and write the code to register all languages in `_numeral.js` using your new `languages.js` version.
Where's the best place to put it ? Line 29 (https://github.com/fbaligand/kibana/blob/f6d8f5e251cd1c2e8d0aafb1fa9a65d2150557c4/src/core_plugins/kibana/common/field_formats/types/_numeral.js#L29) ?